### ISSUE TITLE
Not working for begin environment

### ISSUE THREAD

**Issue by arpit15** [io12/org-fragtog#18]
The package works for $ environment. However, it doesn't work for `begin` environment. I am able to toggle the preview using keybinding `C-c C-x C-l` for both cases. 

I tried following block in org mode
```latex
\begin{align}
\label{myeq}
x=10
\end{align}
```

Thanks for creating this package. I love this package.

**Comment by io12** [comment]
Glad you like my package! Unfortunately, I'm not sure if I can reproduce this bug. The example block you posted works for me. I opened a new empty org-mode buffer, typed
```latex
\begin{align}
\label{myeq}
x=10
\end{align}
A
```
and the fragment appeared. I needed to type the `A` (or any non-space character) at the end because apparently org-mode thinks the context of spaces after a LaTeX fragment is also part of the LaTeX fragment (although I think I know how to fix this if this is the issue you're talking about).
